### ISSUE TITLE
[feature-request] `native_parameters()`: posterior draws of model parameters on the native scale

### ISSUE THREAD

**Issue by GidonFrischkorn** [popov-lab/bmm#388]
bmm samples all parameters on their link scales (`log` for kappa/c/s, logit/softmax for mixture weights, `tan_half` for mu). Users who want the parameter values on their native scale — the thing they report in a paper — currently do manual inverse-link arithmetic on internal nlpar/dpar names. `conditional_effects(scale = "native")` shows the machinery exists, but it only produces plots.

I propose a draws-returning function:

```r
native_parameters(
  fit,
  newdata = NULL,        # defaults to the unique observed design cells
  pars = NULL,           # defaults to all estimated + fixed parameters
  re_formula = NULL,
  scale = c("native", "sampling"),
  ndraws = NULL,
  summary = FALSE
)
```

Return: a long `draws_df` with `.draw`, the grid columns, `parameter`, `value`. Any contrast is then arithmetic on draws (`kappa[ss==8] - kappa[ss==1]`), with the baseline explicit in the grid — no delta method, no ambiguity.

Feasibility: `brms::posterior_linpred(fit, newdata, nlpar =, re_formula =)` does the heavy lifting; bmm already stores the parameter-to-nlpar map and the links, and `conditional_effects.bmmfit()` is a working prototype of the pipeline.

Four design constraints that follow from the math, not from taste:

- [ ] **Transform draws, then summarise.** Quantiles are equivariant under monotone inverse links; means are not (Jensen). A summary-level toggle would be wrong; a draws-level function is exact for medians and CIs.
- [ ] **The inverse link is a property of the model, not of a single parameter.** Mixture weights go through a softmax that is joint across parameters and set-size-dependent; m3 maps activations to probabilities via the choice rule. Thus the transform needs a model-level S3 hook (e.g. `native_transform.bmmodel(model, linpred, data)`), not a per-parameter link lookup. This dovetails with #189.
- [ ] **`re_formula` semantics must be documented up front.** `re_formula = NULL` gives subject-specific parameters; `re_formula = NA` gives the *median* subject (inverse link of the population-mean predictor); the population *mean* native parameter requires marginalising over the RE distribution (`sample_new_levels = "gaussian"`). These diverge substantially at the between-subject SDs typical for WM data.
- [ ] **Bounded contract.** Parameters fixed to constants appear at their constant value; trial-level derived quantities (IMM response probabilities, m3 category probabilities) are out of scope — that is `posterior_epred()` territory. When a user-written non-linear formula makes the parameter map unresolvable (cf. #378), error informatively rather than guess.

Explicitly *not* proposed: a `scale = "native"` argument on the `summary()` coefficient table. Under the idiomatic `kappa ~ 0 + set_size` the coefficients are cell means and `exp()` would be correct — but under `kappa ~ set_size` or any continuous predictor, `exp(b)` is a multiplicative factor, not a kappa, and the transform is undefined for `sd_` and `cor_` rows. A summary toggle would be right in every vignette and silently wrong in users' own analyses. The `summary()` footer could instead point to `native_parameters()`.

Once the transform hook exists, `marginaleffects` support becomes mostly wiring — I would sequence that as a follow-up.
